api/src/main/java/org/pmiops/workbench/db/dao/InstitutionDao.java

```diff
@@ -6,4 +6,6 @@
 
 public interface InstitutionDao extends CrudRepository<DbInstitution, Long> {
   Optional<DbInstitution> findOneByShortName(final String shortName);
+
+  Optional<DbInstitution> findOneByDisplayName(final String displayName);
```

Comment:
Are we using this anywhere?

Reply:
not yet - my update to PopulateOpsUserAffiliations will